src/main/java/frc/robot/Robot.java

```diff
@@ -156,7 +156,7 @@ private void configureBindings() {
     hardware
         .driverController
         .rightTrigger()
-        .onTrue(robotCommands.confirmShotCommand())
+        .onTrue(robotCommands.speakerOrAmpCommand())
         .onFalse(robotCommands.stopShootingCommand());
     hardware
         .driverController
```

Comment:
wrong behavior on intake button release, should do nothing for now, and in the future it'll do assisted intake